README.md

```diff
@@ -100,38 +115,70 @@ Also, make sure your firewall is configured to allow inbound connections to port
 
 ### Binaries (Windows/Linux/macOS)
 
-Binary releases are provided for common operating systems and architectures:
+Binary releases are provided for common operating systems and architectures.
+The easiest method is to download Decrediton from the link below, which will include dcrd.
+Advanced users may prefer the Command-line app suite, which includes dcrd and dcrwallet.
 
 https://decred.org/downloads
 
+* How to verify binaries before installing: https://docs.decred.org/advanced/verifying-binaries/
+* How to install the CLI Suite: https://docs.decred.org/wallets/cli/cli-installation/
+* How to install Decrediton: https://docs.decred.org/wallets/decrediton/decrediton-setup/
+
 ### Build from source (all platforms)
 
-Building or updating from source requires the following build dependencies:
+<details><summary><b>Install Dependencies</b></summary>
 
 - **Go 1.13 or 1.14**
 
   Installation instructions can be found here: https://golang.org/doc/install.
-  It is recommended to add `$GOPATH/bin` to your `PATH` at this point.
+  Ensure Go was installed properly and is a supported version:
+    ```sh
+    go version
+    go env GOROOT GOPATH
+    ```
+  NOTE: `GOROOT` and `GOPATH` must not be on the same path. Since Go 1.8 (2016), `GOROOT` and `GOPATH` are set automatically, and you do not need to change them. However, you still need to add `$GOPATH/bin` to your `PATH` in order to run binaries installed by `go get` and `go install` (On Windows, this happens automatically).
```

Comment:
This is not wrapped consistently as the rest of the file.

Reply:
What's the col limit? Most lines are <=82, but the badges at the beginning go to 131.

Reply:
It's typically 80, but it's not super strict, particularly for links that can't be split.

Reply:
Fixed